packages/validator/src/convertor.ts

```diff
@@ -0,0 +1,98 @@
+import { Log, Result } from './types';
+import * as logger from './utils/logger';
+import fs from 'fs';
+import { processSpecs } from './processor';
+
+const oas2ois = 'templates/OAS2OIS.json';
+const ois2cs = 'templates/OIS2C&S.json';
+
+function invalidConversionMessage(from, to): Log {
+  return logger.error(`Conversion from ${from} to ${to} is not valid conversion`);
+}
+
+/**
+ * Converts specification from provided file into format the template specifies
+ * @param specsPath - specification file to convert, root must be an object (not an array)
+ * @param templatePath - template json file
+ * @returns array of messages and converted specification
+ */
+export function convert(specsPath: string | undefined, templatePath: string | undefined): Result {
+  if (!specsPath || !templatePath) {
+    return { valid: false, messages: [logger.error('Specification and template file must be provided')], output: {} };
+  }
+
+  let template, specs;
+
+  try {
+    template = fs.readFileSync(templatePath);
+  } catch (e) {
+    return { valid: false, messages: [logger.error(`Unable to read file ${templatePath}`)], output: {} };
+  }
+
+  try {
+    specs = fs.readFileSync(specsPath);
+  } catch (e) {
+    return { valid: false, messages: [logger.error(`Unable to read file ${specsPath}`)], output: {} };
+  }
+
+  return convertJson(specs, template);
```

Comment:
nit: since the function is called `convertJson`, I think it would make more sense to convert `specs` and `template` to json before passing them to this function.

Reply:
That really makes much more sense, updated in 77fdfd2
Just a side note about why to call `JSON.parse` on variables in `readmeExamples.test.ts`, instead of directly making the variables object instead of string. This is so they match regular expression from `README.md` one to one (the `\` problem)